src/MeetingsSDKAdapter.js

```diff
@@ -194,6 +211,116 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to mute audio
+   * @memberof MeetingsSDKAdapter
+   */
+  muteAudioMeeting(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting
+      .muteAudio()
+      .then(() => {
+        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MEDIA_TYPE_LOCAL_MUTE_AUDIO});
```

Comment:
Can you just send the control instead of creating a type? 
```suggestion
        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MUTE_AUDIO_CONTROL});
```

OR

```suggestion
        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {control: MUTE_AUDIO_CONTROL});
```